packages/angular/src/directives/control-value-accessors/radio-value-accessor.ts

```diff
@@ -19,9 +18,12 @@ export class RadioValueAccessorDirective extends ValueAccessor {
     super(injector, el);
   }
 
-  // TODO(FW-2827): type (HTMLIonRadioElement and HTMLElement are both missing `checked`)
   @HostListener('ionSelect', ['$event.target'])
   _handleIonSelect(el: any): void {
+    /**
+     * The `el` type is any to access the `checked` state property
+     * that is not exposed on the type interface.
```

Comment:
Why is it not exposed? Are there plans to expose it?

Reply:
Sorry just seeing this feedback! This decision was made before the current team in the framework wrappers repository. 

If I was to guess, it was `protected` since the element reference was only ever used internal to the framework wrapper class. They likely did not plan for what happens when extending the framework wrapper to append additionally functionality that requires the element reference.

We could absolutely change this, we would just need to do a design document to de-risk the assumptions.

Reply:
I deleted my original reply, if you received the alert for it. I should have looked at the referenced file for this comment.

The `checked` property on `ion-radio` is an internal API that we don't want to expose to developers. 

I'm not sure why historically they used a `@State` property for this vs. doing:
```ts
/**
 * @internal 
 */
@Prop() checked = false;
```

We could create a tech debt ticket to investigate this more if you think it would be helpful!

Reply:
It could be worth looking into. Maybe start by asking the team in case anyone remembers.